appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/page-components/reject.mapper.js

```diff
@@ -217,26 +231,49 @@ export function rejectCheckYourAnswersPage(appealDetails, comment, rejectionReas
 /**
  * @param {Representation} comment
  * @param {RepresentationRejectionReason[]} rejectionReasonOptions
+ * @param {import('@pins/express').Session} session
  * @param {{ optionId: number, message: string }} [error]
  * @returns {import('../../../../../appeals/appeals.types.js').CheckboxItemParameter[]}
  */
 export function mapRejectionReasonOptionsToCheckboxItemParameters(
 	comment,
 	rejectionReasonOptions,
+	session,
 	error
 ) {
 	const rejectionReasons = comment.rejectionReasons || [];
 	const rejectionReasonMap = new Map(rejectionReasons.map((reason) => [reason.id, reason]));
 
+	const selectedReasons = (() => {
+		const value = session.rejectIpComment?.rejectionReason;
+		if (!value) {
+			return [];
+		}
+
+		return Array.isArray(value) ? value : [value];
```

Comment:
I feel like I made an until that did this recently. I think I would have called it ensureArray or coerceToArray? I may be thinking of front office though

Reply:
I did think of making a util but then since it's just a simple one-line ternary I thought it might introduce more obfuscation than it's worth.